lib/adf_dataset.py

```diff
@@ -245,14 +279,50 @@ def load_dataset(self, fils):
         return ds
 
 
-    def load_da(self, fils, variablename):
+    # Load DataArray
+    def load_da(self, fils, variablename, **kwargs):
+        """Return xarray DataArray from files(s) w/ optional scale factor, offset, and/or new units"""
         ds = self.load_dataset(fils)
         if ds is None:
             warnings.warn(f"ERROR: Load failed for {variablename}")
             return None
         da = (ds[variablename]).squeeze()
-        if variablename in self.adf.variable_defaults:
-            vres = self.adf.variable_defaults[variablename]
-            da = da * vres.get("scale_factor",1) + vres.get("add_offset", 0)
-            da.attrs['units'] = vres.get("new_unit", da.attrs.get('units', 'none'))
+
+        da = da * kwargs["scale_factor"] + kwargs["add_offset"]
```

Comment:
I think we probably just need to make this a bit more robust with something like
`scale_factor = kwargs.get('scale_factor', 1)`
`add_offset = kwargs.get('add_offset', 0)`
Just in case it gets used without the kwargs being passed.

Reply:
Actually, maybe use `get_defaults` to do this?

Reply:
That's what I was attempting to do with `get_defaults`, it should be setting those to the defaults before it becomes the `kwargs`

I have the offset set to 0 and scale factor to 1 initially, then the check `if variablename in res` should grab the values and if not it will default to 0/1 respectively with `vres.get`. 

And if the variable is not in `self.adf.variable_defaults`, the original values for `new_unit`, `add_offset` , and `scale_factor` will take on the original set values of none, 0, and 1.